src/app/projects/projects-list/projects-list.component.ts

```diff
@@ -81,4 +88,54 @@ export class ProjectsListComponent implements OnInit, OnDestroy {
   changePage(page: PaginationEvent) {
     this.projectService.changePage(page.currentPage);
   }
+
+  saveSearchResultsAsTsv() {
+    const columns = {
+      uuid: 'Unique Key',
+      date: 'Date added',
+      title: 'Project Title',
+      publications: 'Publications',
+      authors: 'Authors',
+      organs: 'Organs',
+      technologies: 'Technologies',
+      cellCount: 'Cell count',
+      enaAccessions: 'ENA',
+      arrayExpressAccessions: 'Arrayexpress',
+      geoAccessions: 'GEO',
+      egaStudiesAccessions: 'EGA',
+      dcpUrl: 'HCA Data Portal URL'
+    };
+    const tsvArray = this.projectsAsTsvArray(this.filteredProjects, columns);
+    const tsvString = tsvArray.join('\r\n');
+    const blob = new Blob([tsvString], {type: 'text/tab-separated-values' });
+    saveAs(blob, 'HcaCatalogueExport.tsv');
+  }
+
+  private projectsAsTsvArray(projects: Project[], columns: object) {
+    const tsvArray = projects.map(
+      (project: Project) => this.projectAsTsvString(project, Object.keys(columns))
+    );
+    tsvArray.unshift(Object.values(columns).join('\t'));
+    return tsvArray;
+  }
+
+  private projectAsTsvString(project: Project, keys) {
+    return keys.map(key => this.flattenProjectField(key, project[key])).join('\t');
+  }
+
+  private flattenProjectField(key: string, value) {
+    if (value === null || value === false) {
```

Comment:
I think this should be `!value && value !== 0` since this should cover undefined too